pages/installation/docker.rst

```diff
@@ -452,6 +452,13 @@ For a stateful set, the name of the first pod in a cluster always ends with ``-0
 * Verify that the pod name ends in ``-0``
 * Set ``is_master=true`` for this container
 
+Nomad automatic master selection
+================================
+
+Running Graylog in Nomad Cluster opens the same challenge to set ``is_master=true`` only for one node in the cluster. That the container runs on Nomad can be identified with an environmetal check on ``NOMAD_ALLOC_INDEX``. Should the variable be set and is ``0`` the container will set the variable ``is_master=true``. If the variable is set but not ``0 `` it will set ``is_master=false``.
```

Comment:
Darf ich eine Änderung vorschlagen?

To verify the container running within a Nomad cluster, check the environment variable ``NOMAD_ALLOC_INDEX``. If it exists and set to ``0``, the container will set the variable ``is_master`` to ``true``, otherwise to ``false``.